src/main/java/org/radarbase/appserver/service/UserService.java

```diff
@@ -164,7 +169,10 @@ public FcmUserDto saveUserInProject(FcmUserDto userDto) {
       User newUser = userConverter.dtoToEntity(userDto).setProject(project.get());
       // maintain a bi-directional relationship
       newUser.getUsermetrics().setUser(newUser);
-      return userConverter.entityToDto(this.userRepository.save(newUser));
+      User savedUser = this.userRepository.save(newUser);
+      // Generate schedule for user
+      this.scheduleService.generateScheduleUsingProjectIdAndSubjectId(savedUser.getProject().getProjectId(), savedUser.getSubjectId());
```

Comment:
Do we need to do this in the update user method too (for example if attribute, timezone or enrolment date is changed)?

Reply:
Yup made the changes.